lightly/api/api_workflow_compute_worker.py

```diff
@@ -489,3 +496,26 @@ def _snake_to_camel_case(snake: str) -> str:
     return components[0] + "".join(
         component.title() for component in components[1:]
     )
+
+
+def _validate_config(cfg: Optional[Dict[str, Any]], obj: Any) -> None:
+    """Validates that all keys in cfg are legit configuration options. """
+
+    if cfg is None:
+        return
+
+    for key, item in cfg.items():
+        if not hasattr(obj, key):
+            error_msg = f"Option `{key}` does not exist!"
+            if hasattr(type(obj), "swagger_types"):
```

Comment:
Suggestion: Add a comment that we suggest the closest possible match to the user for swagger classes. Can be here or in the function docstring.

Reply:
Optional: Maybe this should be an assert at the beginning of the function, or we can type `obj` with the union of allowed types. The motivation is that if we once switch away from swagger generator, the suggestion functionality silently stops working.

Edit: This would not work easily with recursive calls.

Reply:
I solved it by raising a type error when swagger types is missing.